sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/models/DtmfTone.java

```diff
@@ -68,8 +68,72 @@ public static DtmfTone fromString(String name) {
         return fromString(name, DtmfTone.class);
     }
 
+    /** @return known Tone values in char. */
+    public char toChar() {
+        return convertToChar();
+    }
+
     /** @return known Tone values. */
     public static Collection<DtmfTone> values() {
         return values(DtmfTone.class);
     }
+
+    private char convertToChar() {
+        String toneValue = this.toString();
+        char toneInChar = ' ';
+        switch (toneValue) {
+            case "zero":
+                toneInChar = '0';
+                break;
+            case "one":
+                toneInChar = '1';
+                break;
+            case "two":
+                toneInChar = '2';
+                break;
+            case "three":
+                toneInChar = '3';
+                break;
+            case "four":
+                toneInChar = '4';
+                break;
+            case "five":
+                toneInChar = '5';
+                break;
+            case "six":
+                toneInChar = '6';
+                break;
+            case "seven":
+                toneInChar = '7';
+                break;
+            case "eight":
+                toneInChar = '8';
+                break;
+            case "nine":
+                toneInChar = '9';
+                break;
+            case "a":
+                toneInChar = 'a';
+                break;
+            case "b":
+                toneInChar = 'b';
+                break;
+            case "c":
+                toneInChar = 'c';
+                break;
+            case "d":
+                toneInChar = 'd';
+                break;
+            case "pound":
+                toneInChar = '#';
+                break;
+            case "asterisk":
+                toneInChar = '*';
+                break;
+            default:
+                break;
```

Comment:
wouldn't default mean that we receive some weird thing? should we throw an error instead?

Reply:
currently it return a empty char, will look at error throws

Reply:
throw exception cause a chains of problems that every place use this function would need exception handling. On the other hand, the DtmfTone is a result parsed from the returned schema, shouldn't the error be handled in the initial DtmfTone construction other than string converstion?

Reply:
will make the default to return empty char.